src/integrationTest/java/uk/gov/hmcts/reform/sendletter/e2e/BaseTest.java

```diff
@@ -150,4 +154,12 @@ private boolean isValidPdf(boolean isEncryptionEnabled, File file) {
         }
         return true;
     }
+
+    @EnableConfigurationProperties(FtpConfigProperties.class)
+    static class TestConfiguration {
```

Comment:
Not sure why we need this ?

Reply:
in config values are empty and library spat out "empty file" error when i tried to run these tests. I can try removing this but not sure how it works in the pipeline